backends/qualcomm/utils/utils.py

```diff
@@ -184,6 +227,85 @@ def capture_program(
     return edge_ep
 
 
+def from_context_binary(ctx_path: str, op_name: str):
+    def build_graph(inputs, outputs):
```

Comment:
We construct 4 graphs and each graph has one custom op represented a pre-generated context binary. We will ship 4 PTEs onto device and leverage spill-fill to reduce memory footprint.
Since we could not know the graph IO in advanced, all the information is retrieved in runtime by QNN SDK. To avoid operator conflict, the API expects user to manage their own name.